design-system/Header/HeaderDemo.tsx

```diff
@@ -19,7 +19,7 @@ export const HeaderDemo = memo(function HeaderDemo() {
       {/* Basic header with title */}
       <Header
         title="Basic Header"
-        backgroundColor={theme.colors.background.surface}
+        backgroundColor={theme.colors.background.surfaceless}
```

Comment:
Do you find the demo useful? I feel maybe we should delete them. 🤔 

Just some extra code for nothing IMO